src/app/components/BookCard/index.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 import { Text, View, Image } from 'react-native';
+import { Book } from '@interfaces/book';
+import bookPlaceholderCover from '@assets/book_placeholder_cover.png';
 
 import styles from './styles';
 
 interface Props {
-  title: string;
-  author: string;
-  coverImageURI: string;
+  bookData: Book;
 }
```

Comment:
`interface Props extends Book {}` for now

Reply:
Cool. I didn't know that.